### DISCUSSION TITLE
怎么删除创建的用户

### DISCUSSION THREAD

**Discussion by dabu680** [apolloconfig/apollo#3590] [Q&A]
在apollo-portal中创建的用户怎么去删除呢?

**Comment by nobodyiam** [comment]
目前在页面上还没有提供删除功能哈，欢迎 PR。
可行的 workaround 是：
1. 在数据库中更新该 User 记录的 Enabled  值为 0
2. 在页面上修改该用户的密码

**Reply by renyp-veteran** [reply]
如果只在 portalDB中的user表中，通过sql删除指定用户，会不会有其他权限关联表找不到该用户，导致apollo服务启动失败？

**Reply by renyp-veteran** [reply]
或者，能否提供，在portalDB库中，删除指定用户的可行方案（例如：依次删除user、userrole、rolepermission的执行顺序）

**Reply by nobodyiam** [reply]
不建议删除用户，会导致该用户相关的数据如更改历史、发布历史等数据丢失，建议的方式是禁用该用户，原回答中已更新

**Comment by Anilople** [comment]
Duplicate of https://github.com/apolloconfig/apollo/issues/3383
